app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  def postedat
```

Comment:
can you rename this to something a tad more readable, perhaps follow `snake_case` 👀